README.md

```diff
@@ -1,20 +1,19 @@
 
-# TinyMCE Web Component
+# Official TinyMCE Web Component
```

Comment:
Are we?

Reply:
An interesting question. I would assume anything publicly released by Tiny is "official". I mainly copied the title structure from the react repo.

Reply:
its the official preview of whats to come, full wc support  :)

Reply:
So should I stick "Preview" in the title? "Official TinyMCE Web Component Preview"?

Reply:
If it's a preview, then it shouldn't be version `1.0` and should either be `0.x` or a prerelease though.